test/token.js

```diff
@@ -20,4 +24,18 @@ contract('TestToken', _accounts => {
     let erc20TokenContract = await ValidatorSetContract.instance.methods.erc20TokenContract().call();
     assert.equal(erc20TokenContract.valueOf(), instance.address);
   });
+
+  it('should mint tokens', async () => {
+    assert(accounts.length > 0, 'no accounts to carry on the test');
+    let instance = await Token.deployed();
+    const increment = new BN(1);
```

Comment:
Are you minting only `1 wei` here?

Reply:
#24 does this better. If it is fixed and merged, this PR can close.